### PR TITLE
Update install docs

### PR DESCRIPTION
**Pull request by SimonBiggs**
After running through a PyMedPhys install with someone I have improved the getting started docs.

### CODE REVIEW

File: lib/pymedphys/docs/lib/howto/get-started.rst
Comment (Matthew-Jennings) [inline lib/pymedphys/docs/lib/howto/get-started.rst:29]: No biggie, but could instead suggest a command that doesn't require a subsequent exit? Something like `python --version`?

File: lib/pymedphys/docs/lib/howto/get-started.rst
Comment (Matthew-Jennings) [inline lib/pymedphys/docs/lib/howto/get-started.rst:77]: Maybe make clear the right clicking = pasting? E.g.

"You can copy this command directly into a Windows command prompt. You can _Right Click_ to paste."

**Comment by SimonBiggs** [comment]
Yup, I like those suggestions.

File: lib/pymedphys/docs/lib/howto/get-started.rst
Comment (pchlap) [inline lib/pymedphys/docs/lib/howto/get-started.rst:78]: Within our network, we not only have SSL issues but also need to specify a proxy server for pip to use (I'm guessing this is common on health networks). This can be done via the command line, but hard to provide specific guidance because the user would need to know the proxy server in their network. Best to just leave it out here? Or if you like I could prepare a snippet to put in here with the command and explaining they need to find out the proxy server.

File: lib/pymedphys/docs/lib/howto/get-started.rst
Comment (pchlap) [inline lib/pymedphys/docs/lib/howto/get-started.rst:21]: I'm curious, does this way of installing python require admin rights? Just thinking that for our clinical physicists this could be the first stumbling block...

**Review by pchlap — APPROVED** [review]
This is a great concise simple guide for getting started.

File: lib/pymedphys/docs/lib/howto/get-started.rst
Comment (SimonBiggs) [reply lib/pymedphys/docs/lib/howto/get-started.rst:78]: > Or if you like I could prepare a snippet to put in here with the command and explaining they need to find out the proxy server.

@pchlap that would be amazing. Having an example here that someone can use to fill in the blanks for their own system would be significantly better than having no guidance at all.

Comment (pchlap) [reply lib/pymedphys/docs/lib/howto/get-started.rst:78]: Will do, I'll add it directly to the branch of this pull request? Or rather a fresh branch?

File: lib/pymedphys/docs/lib/howto/get-started.rst
Comment (SimonBiggs) [reply lib/pymedphys/docs/lib/howto/get-started.rst:21]: I'm actually not sure... I suspect updating the path length beyond 260 would certainly need admin rights... and this Windows Store method installs Python in a "very nested" directory. So much so that Windows store method with a 260 character path limit means that some of pymedphys' dependencies can't install.

I do notice though that this max path limit is recommended within the Python install docs:

https://docs.python.org/3/using/windows.html#removing-the-max-path-limitation

File: lib/pymedphys/docs/lib/howto/get-started.rst
Comment (SimonBiggs) [reply lib/pymedphys/docs/lib/howto/get-started.rst]: Implemented :slightly_smiling_face:

File: lib/pymedphys/docs/lib/howto/get-started.rst
Comment (SimonBiggs) [reply lib/pymedphys/docs/lib/howto/get-started.rst]: Implemented :slightly_smiling_face:

File: lib/pymedphys/docs/lib/howto/get-started.rst
Comment (SimonBiggs) [reply lib/pymedphys/docs/lib/howto/get-started.rst:78]: If it's just a small addition happy to have it included directly into this PR. But really whichever you prefer.

Comment (pchlap) [reply lib/pymedphys/docs/lib/howto/get-started.rst:78]: Cool, I've popped something into this PR. Hopefully that's suitable?

File: lib/pymedphys/docs/lib/howto/get-started.rst
Comment (pchlap) [reply lib/pymedphys/docs/lib/howto/get-started.rst:21]: Yep, fair enough. Let's leave it for now. But I'll try to run through this at some point with one of our clinical physicists and see how far they get. Perhaps we'll just have to add at the top that this guide assumes they are working on a device where they have admin rights.

Comment (SimonBiggs) [reply lib/pymedphys/docs/lib/howto/get-started.rst:21]: > But I'll try to run through this at some point with one of our clinical physicists and see how far they get

That'd be amazing

> Perhaps we'll just have to add at the top that this guide assumes they are working on a device where they have admin rights.

I think if there is a simple approach that doesn't need admin rights we should opt to go that way. It certainly is possible (I didn't have admin rights back at RCCC, I did get the path limit bumped by IT though).

File: lib/pymedphys/docs/lib/howto/get-started.rst
Comment (SimonBiggs) [inline lib/pymedphys/docs/lib/howto/get-started.rst:115]: Yup, perfect. Thanks @pchlap.

**Comment by SimonBiggs** [comment]
@Matthew-Jennings given you left a comment I'll wait until you leave an approval before merging. :slightly_smiling_face:, but I think this is now ready for merge.

**Review by Matthew-Jennings — APPROVED** [review]
LGTM

File: lib/pymedphys/docs/lib/howto/get-started.rst
Comment (SimonBiggs) [reply lib/pymedphys/docs/lib/howto/get-started.rst:21]: Hi @pchlap,

When you do get someone to run through it, it appears the Windows 10 store install approach doesn't add python scripts to the path. So the pymedphys CLI won't be available. PyMedPhys can still be called by running `python -m pymedphys --help`. 

It might be worth us having a "test to make sure the pymedphys CLI is working" section, and if it's not, know that you can use `python -m pymedphys etc...` or you can use pymedphys directly by adding Scripts to the path (provide instructions on how to do this).

**Pull request merged**
